frontend/webpack.config.ts

```diff
@@ -163,12 +163,15 @@ module.exports = function (env: any, argv: { hot?: boolean; mode: string | undef
         '/multicloud/metrics',
         '/multicloud/login',
         '/multicloud/logout',
+        '/multicloud/observability',
         '/multicloud/operatorCheck',
+        '/multicloud/prometheus',
         '/multicloud/proxy/search',
         '/multicloud/aggregate',
         '/multicloud/username',
         '/multicloud/userpreference',
         '/multicloud/version',
+        '/multicloud/virtualmachineinstances',
```

Comment:
Need /multicloud/virtualmachines as well.
/virtualmachines is used for - start, stop, restart
/virtualmachineinstances is used for - pause & unpase